test/test_dependencies.jl

```diff
@@ -0,0 +1,54 @@
+using Mimi
+
+Pkg.add("InfoZIP")
```

Comment:
Package dependencies are specified in a REQUIRE file in the test folder. Just add a text file named REQUIRE to the test folder that has one line with the content ``InfoZIP`` and the package manager will install the InfoZIP package automatically before it tries to run the tests. Then add a ``using InfoZIP`` line here.